product_warranty/wizard/product_warranty.py

```diff
@@ -0,0 +1,50 @@
+from odoo import Command,_, api, fields, models
+
+
+class ProductWarranty(models.TransientModel):
+    _name = 'product.warranty'
+    _description = "Add warranty"
+
+    sale_order_id = fields.Many2one(
+        'sale.order',
+        string="Order",
+        required=True,
+    )
+    warranty_ids = fields.One2many(
```

Comment:
```suggestion
    warranty_line_ids = fields.One2many(
```
well `line_ids` could work too. nvm